src/lib/components/molecules/slope-chart/index.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useMemo, useRef, useState, useLayoutEffect } from 'react'
+import { positionLabels, scaleLinear } from './slope-chart-util'
+
+export const SlopeChart = ({
+  id,
+  slopeChartItems,
+  padding = { left: 20, right: 24, top: 20, bottom: 16 },
+  domain,
+  prevValKey,
+  nextValKey,
+  abbrKey,
+  nextLabel,
+  prevLabel,
+}) => {
+  const wrapperRef = useRef(null)
+  const [width, setWidth] = useState(0)
+  const contentSize = Math.floor(width - padding.left - padding.right)
+  const height = contentSize + padding.top + padding.bottom
+  const yScale = scaleLinear(domain, [contentSize, 0])
+  const show = width > 0
+
+  useLayoutEffect(() => {
```

Comment:
Given that this effect installs a listener on the window, I think it should use `useEffect` instead of `useLayoutEffect`, i.e. we only want the listener to be added once when the component is created. Although arguably the window size listener could be extracted into a `useWindowSize` hook, and then setting the width based on the wrapperRef could still live in `useLayoutEffect`.